airbyte-integrations/connectors/source-freshdesk/integration_tests/configured_catalog.json

```diff
@@ -0,0 +1,136 @@
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "agents",
+        "json_schema": {},
+        "supported_sync_modes": ["incremental"],
```

Comment:
A very strange catalog. Why is `supported_sync_modes` a `incremental`, can't it be the same as `full_refresh`?